geode-core/src/main/java/org/apache/geode/distributed/internal/InternalLocator.java

```diff
@@ -756,9 +757,9 @@ private void startClusterManagementService() throws IOException {
         new ImmutablePair<>(HttpService.CLUSTER_MANAGEMENT_SERVICE_CONTEXT_PARAM,
             clusterManagementService);
 
-    if (Boolean.getBoolean(ClusterManagementService.FEATURE_FLAG)) {
-      logger.info("System Property {}=true Geode Management API is enabled.",
-          ClusterManagementService.FEATURE_FLAG);
+    if (distributionConfig.getEnableManagementRestService()) {
+      logger.info("Geode Property {}=true Geode Management Rest Service is enabled.",
+          ConfigurationProperties.ENABLE_MANAGEMENT_REST_SERVICE);
       internalCache.getHttpService().ifPresent(x -> {
```

Comment:
Should we log a warning here if the http service is not present or if cluster management service is disabled?

Reply:
if the http service is disabled, there is a message saying "http service is not enabled with port = 0", and none of these messages will show up.